src/components/TodoModal/TodoModal.tsx

```diff
@@ -15,33 +43,52 @@ export const TodoModal: React.FC = () => {
               className="modal-card-title has-text-weight-medium"
               data-cy="modal-header"
             >
-              Todo #2
+              {`Todo #${id}`}
             </div>
 
-            {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
             <button
               type="button"
+              aria-label="clear"
               className="delete"
               data-cy="modal-close"
+              onClick={() => {
+                onsetSelectedTodo(null);
+              }}
+
             />
           </header>
 
-          <div className="modal-card-body">
-            <p className="block" data-cy="modal-title">
-              quis ut nam facilis et officia qui
-            </p>
+          {errorUser
+            ? (
+              <div className="modal-card-body">
+                <p className="block" data-cy="modal-title">
```

Comment:
```suggestion
              <div className="modal-card-body">
                <p className="block" data-cy="modal-title">
```

This part is repeated twice. Optimize code.